org.btchd.btchd-qt.json

```diff
@@ -0,0 +1,88 @@
+{
+  "id": "org.btchd.btchd-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "20.08",
+  "command": "btchd-qt",
+  "finish-args": [
+    /* Xorg support */
+    "--socket=x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network",
+    /* Tor auth cookie */
+    "--filesystem=/run/tor/control.authcookie:ro",
+    "--filesystem=/var/run/tor/control.authcookie:ro"
```

Comment:
what does this do?

Reply:
No idea, got it from [the Bitcoin package](https://github.com/flathub/org.bitcoincore.bitcoin-qt/blob/master/org.bitcoincore.bitcoin-qt.json).

Reply:
Removed it for now to be safe.

Reply:
Update: It's needed to allow some things to be accessed outside the sandbox. [From the wiki](https://docs.flatpak.org/en/latest/manifests.html#finishing). I'll research it more and add it back with the right options.

Reply:
what things?

Reply:
X11 support so it can have a GUI, PulseAudio for sound, OpenGL, and the most important being network access.

Reply:
talking about line 19 and 20.

Reply:
https://github.com/btchd/btchd/blob/master/doc/tor.md